posthog/api/insight.py

```diff
@@ -468,6 +468,11 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
             elif key == "my_last_viewed":
                 if str_to_bool(request.GET["my_last_viewed"]):
                     queryset = self._annotate_with_my_last_viewed_at(queryset).filter(my_last_viewed_at__isnull=False)
+            elif key == "feature_flag":
+                queryset = queryset.filter(
+                    Q(filters__breakdown__icontains=request.GET["feature_flag"])
+                    | Q(filters__properties__values__icontains=request.GET["feature_flag"])
```

Comment:
Since this can be old style dicts without the `values` key, this won't work in all cases.

If this resolves to a string search (I'm unsure), can just do `Q(filters__properties__icontains)`